### DISCUSSION TITLE
Hover animation AND useTransition

### DISCUSSION THREAD

**Discussion by segovo** [pmndrs/react-spring#1351] [Support]
Hey guys, I'm using useTransition to do essentially [this (from the docs).](https://codesandbox.io/embed/1y3yyqpq7q) 

Would it be possible or easy to add a hover which does only a part of that animation, having the cards partially slide before being clicked and fully sliding?

**Comment by joshuaellis** [comment]
I'd look at using a seperate spring for the hover & then `useChain` to get the current pos to transition too. Or maybe it'll work with out `useChain`. You'd have to give it a go, but you'll have to definitely use more than one spring.
